### DISCUSSION TITLE
Arc invite

### DISCUSSION THREAD

**Discussion by fowntain** [BigfootsGS/BigfootsGS.github.io#40] [General]
hey devs! 
I'm the owner of a smaller game website, [Skool](https://skool.world), and I saw you guys have arc! I was hoping to get an invite code so we could use it too, thanks for any help! (P.S. We're also [open source](https://github.com/Skoolgq/skoolgq.github.io)!)

**Comment by Bigfoot9999** [comment]
I think it is like 000000
but arc is pretty pointless, for example, interstellar games (they get about if not as much traffic as us) and they have only made 3$
